VTEX - Antifraud Provider API.json

```diff
@@ -2419,62 +1875,58 @@
                     "name": {
                         "type": "string",
-                        "description": "Item name."
+                        "description": "Item name.",
+                        "example": "Some useful product"
                     },
                     "price": {
                         "type": "number",
-                        "description": "Item price."
+                        "description": "Item price.",
+                        "example": 20.51
                     },
                     "quantity": {
                         "type": "integer",
-                        "format": "int32",
-                        "description": "Quantity of this item in the cart."
+                        "description": "Quantity of this item in the cart.",
+                        "example": 2
                     },
                     "deliveryType": {
                         "type": "string",
-                        "description": "Type of delivery configured by the store."
+                        "description": "Type of delivery configured by the store.",
+                        "example": "Normal"
                     },
                     "deliverySlaInMinutes": {
                         "type": "integer",
-                        "format": "int32",
-                        "description": "Each item can have a separate shipping sla in minutes."
+                        "description": "Each item can have a separate shipping sla in minutes.",
+                        "example": 4320
                     },
                     "categoryId": {
                         "type": "string",
-                        "description": "Category ID for the Item. Configured by each store."
+                        "description": "Category ID for the Item. Configured by each store.",
+                        "example": "111"
                     },
                     "categoryName": {
                         "type": "string",
-                        "description": "Category name for the Item. Configured by each store."
+                        "description": "Category name for the Item. Configured by each store.",
+                        "example": "Electronica"
                     },
                     "discount": {
                         "type": "number",
-                        "description": "Discount applied on item(s) two decimal places."
+                        "description": "Discount applied on item(s) two decimal places.",
+                        "example": 1.99
                     },
                     "sellerId": {
                         "type": "string",
-                        "description": "In case of a marketplace transaction, this is the Id of the seller for this specific item. Otherwise, this is filled with a '1'."
+                        "description": "In case of a marketplace transaction, this is the Id of the seller for this specific item. Otherwise, this is filled with a '1'.",
```

Comment:
```suggestion
                        "description": "In case of a marketplace transaction, this is the ID of the seller for this specific item. Otherwise, this is filled with a '1'.",
```

Reply:
Modificado.